h3/src/tests/request.rs

```diff
@@ -398,20 +419,19 @@ async fn header_too_big_client_error() {
                 }
             );
         };
-        tokio::select! {biased; _ = req_fut => (),_ = drive_fut => () }
+        tokio::join! {req_fut, drive_fut }
     };
 
     let server_fut = async {
         let conn = server.next().await;
-        //= https://www.rfc-editor.org/rfc/rfc9114#section-4.2.2
-        //= type=test
```

Comment:
Why remove this comment? It's used for RFC coverage.

Reply:
if I am correct this does not test the server limit. It tests that our client does not send any requests that extend the server limit.